code/processes/segmentedtickerplant.q

```diff
@@ -83,5 +83,38 @@ init:{
   .stplg.init[string .proc.procname];
  };
 
-// Have the init function called from torq.q
+// Loads the striping.json config file and logs any problems
+configload:{[scpath]
+     @[jsonchecks; 
+       .stpps.stripeconfig:@[{.j.k read1 x};scpath;{.lg.o[`configload;"Failed to load in striping.json file: ",x]}];
```

Comment:
if we fail to read the json file, do we want to continue or should we just exit at that point? (assuming this happens during proc initialisation we can just log as an error I think & torq will exit the process automatically)

Reply:
OK fair enough - as long as everything else works OK even if this fails to load.

Might be worth doing the json load first, then returning early from the `configload` function if it fails rather than continuing & running the `jsonchecks` function which is surely going to fail as well if the json load failed?

Reply:
I believe in testing if the json file failed to load then the output message was logged and no other checks were done. Could be mistaken though.

Reply:
as it's written here, I'm pretty sure `jsonchecks` will get called regardless of whether the json successfully loads